packages/react-query/src/__tests__/useQueries.test.tsx

```diff
@@ -1142,6 +1142,7 @@ describe('useQueries', () => {
         fallbackRender={({ error }) => (
           <div>
             <div>error boundary</div>
+            {/* @ts-expect-error `error` here is actually of type `string` and not `Error` */}
```

Comment:
Can we just reject the queryFn with a new Error instead?

Reply:
[c92a897](https://github.com/TanStack/query/pull/4177/commits/c92a8976fc60c70beb6fc43bf3a1798a19d1e935)